kythe/typescript/README.md

```diff
@@ -1,5 +1,269 @@
 # Kythe indexer for TypeScript
 
+## VName Specification
```

Comment:
The other stuff in the README is like "how to run the tests" which is what I think is a good opener.  I think this belongs below the other stuff in the README, or maybe in a different file.